internal/datastore/context.go

```diff
@@ -66,15 +47,15 @@ func (p *ctxProxy) IsStrictReadModeEnabled() bool {
 }
 
 func (p *ctxProxy) OptimizedRevision(ctx context.Context) (datastore.Revision, error) {
```

Comment:
Is this proxy even used anywhere anymore?

Reply:
Yes, MySQL datastore still uses it.